.github/workflows/ci.yml

```diff
@@ -34,8 +34,11 @@ jobs:
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.0'
+
+          - operating-system: 'ubuntu-20.04'
```

Comment:
why new entry?

Reply:
It is side effect of [this change](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6883#pullrequestreview-1510691407) - when I bumped to SYmfony 6.1 on PHP 8.1 (required version for this release), then there was no test for PHP8.0 anymore. As I said, I'll try to run pipeline with previous matrix, but later today.